applications: serial_lte_modem: fix FTP data reception (#12670)
nrfconnect/sdk-nrf: applications/serial_lte_modem/doc/FTP_AT_commands.rst

The values of the parameters depend on the command string used.
When using the ``put``, ``uput`` and ``mput`` commands, if the ``<data>`` attribute is not specified, SLM enters ``slm_data_mode``.

.. note::

>    The maximum size of the data that can be received when using the commands "ls" and "get" is limited by the size of the internal FTP buffer, which is 2048 bytes.

Reviewer: ```suggestion
   The maximum size of the data that can be received when using the commands ``ls`` and ``get`` is limited by the size of the internal FTP buffer, which is 2048 bytes by default.
```

>    It is ``ftp_data_buf``, defined in :c:file:`src/ftp_c/slm_at_ftp.c`.

Reviewer: ```suggestion
   The size of the buffer can be changed using ``ftp_data_buf``, defined in :c:file:`src/ftp_c/slm_at_ftp.c`.
```

>    If more data than can be handled is received after issuing an "ls" or "get" command, it will be truncated to the maximum supported size.

Reviewer: ```suggestion
   If more data than can be handled is received after issuing an ``ls`` or ``get`` command, the data will be truncated to the maximum supported size.
```
